src/test-reader/build-instructions.js

```diff
@@ -64,7 +64,11 @@ function extendWithTimeout({ treeBuilder, config }) {
 }
 
 function buildGlobalSkipInstruction(config) {
-    const skipBrowsers = env.parseCommaSeparatedValue("HERMIONE_SKIP_BROWSERS");
+    const { value: skipBrowsers, key: envKey } = env.parseCommaSeparatedValue([
+        "TESTPLANE_SKIP_BROWSERS",
+        "HERMIONE_SKIP_BROWSERS",
+    ]);
```

Comment:
In these cases we check both env variables. We need used env key back, so we can define correct skip reason